### PR TITLE
Add DC: GCP CloudAssets

### PR DESCRIPTION
**Pull request by sfc-gh-afedorov**
- creates gcp_cloudassets.py with `ingest` method

tested locally against dev/prod

- created gcp_cloudassets.sql.fmt "pack" template

tested using internal code against dev / prod

minor update to db.py tested by automated tests

### CODE REVIEW

File: src/connectors/gcp_cloudassets.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
+"""GCP Cloud Asset
+Collect GCP Cloud Assets via API and PIPE
```
Comment (sfc-gh-afedorov) [inline src/connectors/gcp_cloudassets.py:2]: not a pipe

File: src/connectors/gcp_cloudassets.py [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+LANDING_TABLES_COLUMNS = {
+    'export': [
+        ('created_on', 'TIMESTAMP_LTZ DEFAULT CURRENT_TIMESTAMP'),
+        ('org_id', 'STRING'),
+        ('raw', 'VARIANT'),
+    ]
+}
+
+
+def exec(client, org_id, prefix):
```
Comment (sfc-gh-afedorov) [inline src/connectors/gcp_cloudassets.py:47]: need better name + docstring

Comment (sfc-gh-afedorov) [reply src/connectors/gcp_cloudassets.py:47]: `init_asset_export_to_gcs`

File: src/connectors/gcp_cloudassets.sql.fmt [hunk 3]
```diff
@@ -0,0 +1,102 @@
+-- ---
```
Comment (sfc-gh-afedorov) [inline src/connectors/gcp_cloudassets.sql.fmt:1]: this is confusing without the SQL generating runner

**Review by sfc-gh-pkommini — APPROVED** [review]

**Pull request merged**